server/matchmaker/matchmaker_queue.py

```diff
@@ -179,17 +179,24 @@ async def find_matches(self) -> None:
         for match in proposed_matches:
             if self.match(match[0], match[1]):
                 matches.append(match)
+                for team in match:
+                    for search in team.get_original_searches():
+                        metrics.matched_matchmaker_searches.labels(
+                            self.name, len(search.players)
+                        ).inc()
```

Comment:
Hmm that's maybe too many levels of indentation. Maybe extract to a method and put it down in the other loop where you're reporting the quality metrics.